build.gradle

```diff
@@ -0,0 +1,89 @@
+import java.util.regex.Pattern
+
+plugins {
+    id 'java-library'
+}
+
+sourceSets {
+    main {
+        java {
+            srcDirs = [ 'src' ]
+            outputDir = file('classes')
+        }
+    }
+    test {
+        java {
+            srcDirs = [ 'test' ]
+        }
+    }
+}
+
+test {
+    useJUnitPlatform()
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'org.apache.ant:ant:1.10.6'
```

Comment:
I believe ant is only needed for compiling. This option should siffice: 
` compileOnly gradleApi() //ANT provided from gradle distribution`